frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1029,9 +1031,10 @@ const SilencesDetailsPage = withFallback(
               breadcrumbs={[
                 {
                   name: t('monitoring~Silences'),
-                  path: namespace
-                    ? `/dev-monitoring/ns/${namespace}/silences`
-                    : '/monitoring/silences',
+                  path:
+                    activePerspective === 'dev'
```

Comment:
I think we should also change the link text to `Alerts` for the dev perspective.

Reply:
Updated the text to `Alerts` for the dev perspective.